projects/ad4630_fmc/common/ad463x_bd.tcl

```diff
@@ -94,8 +94,17 @@ ad_ip_parameter sync_generator CONFIG.PULSE_0_WIDTH [expr int(ceil(double($max17
 
 if {$NO_REORDER == 0} {
 
-  ad_ip_instance spi_axis_reorder data_reorder
-  ad_ip_parameter data_reorder CONFIG.NUM_OF_LANES $NUM_OF_SDI
+ad_ip_instance spi_axis_reorder data_reorder
+ad_ip_parameter data_reorder CONFIG.NUM_OF_LANES $NUM_OF_SDIO
```

Comment:
```suggestion
  ad_ip_instance spi_axis_reorder data_reorder
  ad_ip_parameter data_reorder CONFIG.NUM_OF_LANES $NUM_OF_SDIO
```

Should be indented since it is inside of `{ }` block.